src/main/webapp/about/dev-platform.jsp

```diff
@@ -0,0 +1,113 @@
+                    At the core of most of our projects lies XMPP. XMPP is open, mature, flexible and extensible, making it the protocol of choice for real-time communications over the Internet. It enables the reliable transport of structured data between individuals or applications. The <a href="https://xmpp.org">XMPP Standards Foundation</a> manages the development of the XMPP protocols.
+                </p>
+
+                <p>
+                    Numerous mission-critical business applications use XMPP, including chat and IM, network management and financial trading - and so could yours. With inherent security features and support for cross-domain server federation, XMPP is more than able to meet the needs of the most demanding environments.
+                </p>
+
+                <p>
+                    You can find more information on XMPP in our whitepaper called <a href="../support/articles/whitepaper_xmpp.jsp">"XMPP: The Protocol for Open, Extensible Instant Messaging"</a> or at the website of the <a href="https://xmpp.org">XMPP Standards Foundation</a>.
+                </p>
+
+                <h2 class="larger">Client-oriented Development</h2>
+
+                <p>
+                    <strong><a href="./projects/smack/">Smack</a></strong> is an Open Source XMPP client library for instant messaging and presence. A library for Android and Java, it can be embedded into your applications to create anything from a full XMPP client to simple XMPP integrations such as sending notification messages and presence-enabling devices.
+                </p>
+
+                <h2 class="larger">Server-oriented Development</h2>
+
+                <p>
+                    Our <strong><a href="../projects/openfire/">Openfire</a></strong> server is a real time collaboration (RTC) server licensed under the open source Apache License. Not only can it act as a Turn-key IM solution, it also forms a quite capable development platform.
+                </p>
+
+                <p>
+                    Not few organisations have launched their proprietary instant messaging solution based on the Openfire platform. This often starts with using Openfire as an off-the-shelf component, that is used by purpose-built clients. Openfire's exceptional extensibility, with <a href="https://download.igniterealtime.org/openfire/docs/latest/documentation/index.html#dataprovider-development">fully customizable authentication, group, user and roster providers</a>, a powerful <a href="https://download.igniterealtime.org/openfire/docs/latest/documentation/plugin-dev-guide.html">plugin architecture</a> and excellent <a href="https://download.igniterealtime.org/openfire/docs/latest/documentation/protocol-support.html">protocol support</a> allow for quick and easy server-sided implementations of new features and functionality. Its <a href="../projects/openfire/plugin-archive.jsp?plugin=restAPI">REST API</a> is another common source of focus for integration.
```

Comment:
"Not few" ?

Reply:
Yeah, me 'being creative' when I'm tired is not a great look. Happy to replace this.